bzt/utils.py

```diff
@@ -750,23 +751,25 @@ def default(self, obj):  # pylint: disable=method-hidden
                 else:
                     res[key] = val
             return res
-        elif self.__convertable(obj):
+        elif ComplexEncoder.__convertible(obj):
             return obj.__json__()
         else:
             return None
 
-    def __dumpable(self, obj):
+    @classmethod
+    def __dumpable(cls, obj):
         """
         Re
 
         :param obj:
         :rtype: bool
         """
-        dumpable_types = tuple(self.TYPES + (JSONDumpable,))
```

Comment:
This changes the result completely. Is this intentional change?
Also weird that no unit test has detected this problem

Reply:
could you show me an example when this changes the result? Of course, I mean the result of the method(i.e. isinstance), not value of dumpable_types variable.

Reply:
I don't know if it affects the result of the method. But I believe it's a bad practice to change `tuple of types` into `tuple of mixed types and tuples`, it makes it harder to understand the code.

Reply:
If you want to fix it all, let's do it with separate PR. 
I just want to fix whatever changes in this specific PR.